sqlserver/assets/configuration/spec.yaml

```diff
@@ -162,6 +162,22 @@ files:
       value:
         type: boolean
         example: false
+    - name: include_primary_log_shipping_metrics
+      description: |
```

Comment:
I wonder if we should add an example of this to these docs https://docs.datadoghq.com/database_monitoring/setup_sql_server/selfhosted/?tab=sqlserver2014#example-agent-configurations

is this something that is super common for people to monitor in sqlserver?

Reply:
I will have to modify these docs to discuss the extra permission needed so yeah I might throw in an example